### PR TITLE
feat: parse ruby spec

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1 
Added classes to parse podspec in ruby format.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, review

File: .rultor.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+docker:
```
Comment (dgarus) [inline .rultor.yml]: Do we use `rultor` now?

File: src/main/java/com/artipie/cocoa/metadata/Podspec.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    String license();
+
+    /**
+     * Homepage URL.
+     * @return The URI
+     */
+    URI homepage();
+
+    /**
+     * Packages course, type and value map. Check the
```
Comment (dgarus) [inline src/main/java/com/artipie/cocoa/metadata/Podspec.java]: `course` -> `sourse`?

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
There are minor comments.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected, rultor we do not use now

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

**Pull request merged**